server/migrations/versions/f94e1dc7996d_.py

```diff
@@ -0,0 +1,61 @@
+"""empty message
+
+Revision ID: f94e1dc7996d
+Revises: 
+Create Date: 2021-01-14 01:23:00.656671
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f94e1dc7996d'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('user',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('username', sa.String(), nullable=True),
+    sa.Column('email', sa.String(), nullable=True),
+    sa.Column('password', sa.String(), nullable=True),
+    sa.Column('created_time', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('contest',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('title', sa.String(), nullable=True),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('prize_contest', sa.Integer(), nullable=True),
+    sa.Column('deadline_date', sa.DateTime(), nullable=True),
+    sa.Column('created_time', sa.DateTime(), nullable=True),
+    sa.Column('update_time', sa.DateTime(), nullable=True),
+    sa.Column('contest_creater', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['contest_creater'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('submission',
+    sa.Column('id', sa.Integer(), nullable=False),
```

Comment:
for all IDs, you probably want to make them autoincrement too 
```
sa.Column('id', sa.Ingeter(), autoincrement=True, nullable=False)
```

Reply:
suggestion applied.